std/process.d

```diff
@@ -1266,6 +1266,45 @@ final class Pid
         return _processID;
     }
 
+    /**
+    A simple factory that returns a $(LREF Pid) handle when provided with a
+    global `pid`. This handle can be used to kill processes that haven't been
+    spawned by this process.
+    `Note:` this does not check if the provided `pid` is valid and corresponds
+    to a running process within the operating system.
+
+    Params:
+        pid = a process id
+        desiredAccess = (Windows specific, optional) the desired access to the
+                        process object
+
+    Returns: an instance to a `Pid` that corresponds to the pid within the
+    operating system, or, on Windows, null if opening the process failed.
+    */
+    version (Windows)
+    {
+        static Pid fromSystemPid(int pid, DWORD desiredAccess = PROCESS_ALL_ACCESS)
+        {
+            import core.sys.windows.winbase : OpenProcess;
+            HANDLE _handle = OpenProcess(desiredAccess, false, pid);
+
+            Pid _processHandle;
+            if (_handle !is null)
+            {
+                _processHandle = new Pid(pid, _handle);
+            }
+
+            return _processHandle;
```

Comment:
Most functions of this module will throw on error, I suggest to do the same here.